src/scss/settings/_variables.scss

```diff
@@ -65,6 +65,7 @@ $font-family-mono           : Menlo, Monaco, 'Courier New', monospace;
 //  Global Breakpoints
 // ==========================================================================
 $breakpoints: ko-map-to-em((
+    tiny     : 375px,    // older gen narrow devices
```

Comment:
Which device is this out of interest?

Reply:
375px is a 4.7inch iPhone - 6(s), 7, 8

Reply:
[Useful link](https://www.mydevice.io/):

![screen shot 2018-07-03 at 15 32 08](https://user-images.githubusercontent.com/805184/42226218-4e5dbbb8-7ed6-11e8-98df-0faf37f4e7cc.png)

Wondering if it's worth removing the comment, as apparently iPhone X is also that px width too.

Reply:
I agree, I was naming it to use less than narrow (targeting iPhone 4, 5 widths) - i'll remove :)